### PR TITLE
Added logs to debug reinitialization of WriteConnection Pool and corrected error messages

### PR DESCRIPTION
**Pull request by lakkundipooja**
Added logs to debug reinitialization of Writepool and added proper errors

### CODE REVIEW

**Review by shivacharanms — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]

**Review by amar-shalgar — APPROVED** [review]
LGTM

File: lib-utilities/services/task.go [hunk 1]
```diff
@@ -34,7 +34,7 @@ func CreateTask(sessionUserName string) (string, error) {
 		},
 	)
 	if err != nil && response == nil {
-		log.Error("error: something went wrong with rpc call: " + err.Error())
+		log.Error("Something went wrong with rpc call: " + err.Error())
```
Comment (irappa-pattar) [inline lib-utilities/services/task.go]: change `Something went wrong with rpc call:`  to `rpc error`

**Review by irappa-pattar — CHANGES_REQUESTED** [review]

File: lib-utilities/services/task.go (hunk 1 above)
Comment (lakkundipooja) [reply lib-utilities/services/task.go]: addressed the changes

**Review by irappa-pattar — APPROVED** [review]

**Review by irappa-pattar — APPROVED** [review]

File: lib-persistence-manager/persistencemgr/redis.go [hunk 2]
```diff
@@ -90,14 +90,14 @@ func sentinelNewClient(dbConfig *Config) *redisSentinel.SentinelClient {
 //GetCurrentMasterHostPort is to get the current Redis Master IP and Port from Sentinel.
 func GetCurrentMasterHostPort(dbConfig *Config) (string, string) {
 	sentinelClient := sentinelNewClient(dbConfig)
+	log.Debug("sentinelClient in GetCurrentMasterHostPort : ", sentinelClient)
```
Comment (Bharath-KKB) [inline lib-persistence-manager/persistencemgr/redis.go]: print Addr as string and DB instance as int

**Review by skhan-anjum — APPROVED** [review]

**Review by Bharath-KKB — CHANGES_REQUESTED** [review]
print string and int type by themselves. DO not print aggregates like struct in log messages

File: lib-persistence-manager/persistencemgr/redis.go (hunk 2 above)
Comment (lakkundipooja) [reply lib-persistence-manager/persistencemgr/redis.go]: added MasterIP and masterport in logs to debug

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**